### ISSUE TITLE
[WCAG 2.2 Draft Feedback] Summarized Feedback

### ISSUE THREAD

**Issue by dshoukry** [w3c/wcag#1447]
We've summarized our overall feedback in this issue, and will open up separate issues for each Success Criterion per Alastair's preference. 

Overall summary of comments:

1. As we raised in the 2020 charter response, we recognize that scoping, drafting, and redrafting Success Criteria (SC) many rounds over is quite taxing, and hope our constructive contributions towards the end of an arduous process provide some support.

We saw that there were initially around 13 SC scoped for the 2.2 update. This smaller subset of nine SC seems like an indication that the group held back a good number of guidelines that might not have been totally stable, which is certainly appreciated. 

That being said, can the group share how they handled or attempted to handle some of the other feedback raised by Members in the AGWG 2020 Charter objections?

While these proposed SCs all seem largely beneficial, it would be helpful to understand the research or background for what led to each criteria, beyond anecdotes/described use cases. We still have some of the same concerns raised about WCAG 2.1, such as: 

- the broader implication of applying some of this guidance to everything on the web
- in other cases, expanding the scope of the Web Content Accessibility Guidelines beyond the Open Web Platform 
- the real world benefits to users from highly-specific pixel requirements for focus indicators
- the broad and potentially susceptible asks around help mechanisms and controls' persistent availability

We very much appreciated the group's response and commitments to some of these 2020 charter objection points and are keen to understand how those were incorporated.

2. It would help to provide more references to overlapping WCAG 2.0 and 2.1 guidelines in this documentation. We flagged a few potential opportunities for your review (these will be covered in each SC feedback doc).

**Comment by alastc** [comment]
Hi @dshoukry,

As a group we have responded to the issues raised on each SC, I hope they were satisfactory.

As the chair/editor primarily responsible for managing the issues raised on WCAG 2.2, I wanted to respond to this general issue.

> the broader implication of applying some of this guidance to everything on the web

I think this one is better dealt with in the individual SC threads, but we have been considering the feasibility of each SC for everything from a 1 page simple HTML site, to a multi-national/lingual site with millions of pages. 

The SCs are either scoped to particular scenarios (e.g. providing help, and if you don't provide help it isn't applicable), or take a baseline approach (rather than requiring the ideal approach), or both.


> in other cases, expanding the scope of the Web Content Accessibility Guidelines beyond the Open Web Platform

Our scope is web content, and we have stuck to that. If this was referring to the page-break-navigation SC, that has been removed.


> the real world benefits to users from highly-specific pixel requirements for focus indicators

This comes from needing to set a testable baseline. We can talk about what is ideal (e.g. in the understanding document), but as a standard if we want to require a "visible" focus indicator, that needs to be testable.

In the real world, lack of visible focus indicators is one of the most complained about  issues in usability testing, and [online](https://twitter.com/search?q=%22focus%20indicator%22&src=typed_query&f=top). It is also very common, when my company does audits we have to test the keyboard accessibility, and difficult to see focus indicators come up in most audits.  

Some of these are already captured by [non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html#figure-focus-inner), but some obviously poor indicator are not captured by that.

We worked through examples to establish the factor which affect the visibility. This works for people with regular vision, and they are more noticeable for people with low vision. This was outlined in the [response to the focus-appearance SC](https://github.com/w3c/wcag/issues/1896#issuecomment-883591992).

Whenever you have a requirement on a continuum (e.g. bigger is better), we have to define a baseline. A lot of the input to that is due to the _feasibility_, rather than user-requirement. It has to be implementable and testable. That's where the specific sizes come from, working through many examples to establish feasibility, whilst trying to maintain a good outcome from the user.

> the broad and potentially susceptible asks around help mechanisms and controls' persistent availability

The visible-controls SC has been removed, we could not establish a testable way of knowing what an indicator would be for showing there are controls available.

The SC for consistent help is carefully scoped so that it only applies if you are providing a particular mechanism across a set of web pages.

As there were no replies to the individual SCs, I have to assume that the responses were satisfactory. I'm happy to discuss anything coming out of this issue as well.
